addons/account/account_move_line.py

```diff
@@ -783,6 +783,7 @@ def prepare_move_lines_for_reconciliation_widget(self, cr, uid, lines, target_cu
                 'id': line.id,
                 'name': line.name != '/' and line.move_id.name + ': ' + line.name or line.move_id.name,
                 'ref': line.move_id.ref,
+                'is_reconciled': not line.account_id.reconcile, # for reconciliation with existing entries (eg. cheques)
```

Comment:
Because the lines for this use case are fetched with ('account_id', 'in', [st_line.journal_id.default_credit_account_id.id, st_line.journal_id.default_debit_account_id.id]) and the line that is reconciled is typically in debtors/creditors. Also, that means that you will perform a 'reconciliation' with the line labelled "Payment from X" instead of the "SAJ/year/num"